src/haxelib/client/Main.hx

```diff
@@ -221,8 +221,19 @@ class Main {
 		initSite();
 	}
 
+	function retry<R>(func:Void -> R, numTries:Int = 3) {
+		while (numTries-- > 0) {
+			try {
+				return func();
+			} catch(e:Dynamic) {
+				print('Failed with error: $e');
```

Comment:
I think it's worth to print that we're about to retry the operation and print a success message in case of successful retry.
If user would see `Failed with error` and no success message afterwards, user would think it failed.